shared-bindings/displayio/OnDiskBitmap.c

```diff
@@ -80,16 +80,22 @@
 //|   :param file file: The open bitmap file
```

Comment:
Please add to the documentation comment so that the added functionality will be shown in the online documentation.  You can just copy the format above, adding one line for each of the 4 new arguments you added.  (explaining the function of the three mask values will be difficult, especially for a less technical audience; is it possible that you can get by with *JUST* the single new boolean "dither" flag?  Is there a case where the default will be woefully wrong?)